netbout-web/src/main/java/com/netbout/rest/TkEmVerify.java

```diff
@@ -83,13 +86,26 @@ public TkEmVerify(final Base bas) {
 
     @Override
     public Response act(final RqRegex req) throws IOException {
-        final String invalid = "verification link not valid";
-        final Matcher matcher = TkEmVerify.PATTERN.matcher(
-            TkEmVerify.ENC.decrypt(
-                URLDecoder.decode(req.matcher().group(1), "UTF-8")
-                    .replaceAll(" ", "+")
-            )
-        );
+        String decoded = "";
+        boolean eonpe = false;
+        try {
+            decoded = TkEmVerify.ENC.decrypt(
+                URLDecoder.decode(
+                    req.matcher().group(1), "UTF-8"
+                ).replaceAll(" ", "+")
+            );
+        } catch (final EncryptionOperationNotPossibleException ignore) {
+            eonpe = true;
+        }
+        final String invalid = "verification link not valid.";
+        if (eonpe) {
+            throw new RsForward(
+                new RsFlash(invalid, Level.SEVERE)
+                , HttpURLConnection.HTTP_MOVED_PERM
```

Comment:
@kujtimiihoxha commas should be on the previous line, here and in the line below